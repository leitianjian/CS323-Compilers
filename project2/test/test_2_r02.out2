Error type 2 at Line 8: add function is invoked without deÔ¨Ånition
Error type 5 at Line 8: unmatching types on both sides of assignment operator (=)
