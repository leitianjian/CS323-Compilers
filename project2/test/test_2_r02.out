Error type 2 at line 8: add function is invoked without definition
